config/fastly/snippets/safe_params_list.vcl

```diff
@@ -1,7 +1,7 @@
 import querystring;
 sub vcl_recv {
     # return this URL with only the parameters that match this regular expression
-    if (req.url !~ "/internal/" && req.url !~ "/search/") {
+    if (req.url !~ "/internal/" && req.url !~ "/search/" && req.url !~ "/bulk_show") {
```

Comment:
should this be the entire path `/follows/bulk_show` ? Just in case we add one that shouldn't be considered. 

ps. what if we whitelist `ids` instead?

Reply:
My assumption was that in any bulk_show path we are going to be using `ids[]` or some sort of array param in order to send multiple items so this will cover those situations nicely. 

I have not been able to figure out how to successfully ignore the array syntax for params. I tried a lot of different things for search and none of them worked which is why search ended up on this list.